### PR TITLE
[LastInfoSec] Create the connector (#111)

### PR DESCRIPTION
**Pull request by remydewa**
PR for issue #111

### CODE REVIEW

**Comment by remydewa** [comment]
Connector created. Ready to merge

File: LastInfoSec/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM python:3.7.4-alpine3.10
```
Comment (rhaist) [inline LastInfoSec/Dockerfile]: Please use the current standard `python:3.7-alpine`

File: LastInfoSec/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Copy the connector
+COPY src /opt/opencti-connector-lastinfosec-cti
```
Comment (rhaist) [inline LastInfoSec/Dockerfile]: If there will be no further connector maybe just stick with `opencti-connector-lastinfosec` ?
If okay: replace in all files :)

File: LastInfoSec/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  connector-lastinfosec-cti:
+    image: connector-lastinfosec-cti:0.0.5
```
Comment (rhaist) [inline LastInfoSec/docker-compose.yml]: The container version should be the current OpenCTI major release `:3.2.0`

File: LastInfoSec/src/config.yml.sample [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  id: 'ChangeMe'
+  type: 'EXTERNAL_IMPORT'
+  name: 'lastinfosec-cti'
+  scope: 'cyber threat intelligence'
+  update_existing_data: True
+  log_level: 'info'
+
+config:
+  lastinfosec_url: 'https://api.client.lastinfosec.com/v2/stix2/getlasthour?key='
+  lastinfosec_apikey: 'ChangeMe'
```
Comment (rhaist) [inline LastInfoSec/src/config.yml.sample]: A better format, that is compatible to existing connectors would be:
```
lastinfosec:
  api_url: 'https://api.client.lastinfosec.com/v2/stix2/getlasthour?key='
  api_key: 'ChangeMe'
```

File: LastInfoSec/src/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,4 @@
+pycti
+PyYAML
+urllib3
+python-dateutil
```
Comment (rhaist) [inline LastInfoSec/src/requirements.txt]: Please use versioned requirements like here:
https://github.com/OpenCTI-Platform/connectors/blob/master/amitt/src/requirements.txt

File: LastInfoSec/src/lastinfosec.py [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+
+if __name__ == "__main__":
+    try:
+        lastInfoSecConnector = LastInfoSec()
+        lastInfoSecConnector.run()
+    except Exception as e:
+        traceback.print_exc()
+        time.sleep(10)
+        exit(0)
```
Comment (rhaist) [inline LastInfoSec/src/lastinfosec.py:82]: Please make sure you format with black and check for flake8:
`flake8 --ignore=E,W . && black .`

File: LastInfoSec/Dockerfile (hunk 1 above)
Comment (SamuelHassine) [reply LastInfoSec/Dockerfile]: Agree.

File: LastInfoSec/Dockerfile (hunk 2 above)
Comment (SamuelHassine) [reply LastInfoSec/Dockerfile]: Agree.

File: LastInfoSec/src/config.yml.sample (hunk 4 above)
Comment (SamuelHassine) [reply LastInfoSec/src/config.yml.sample]: Agree.

File: LastInfoSec/src/requirements.txt (hunk 5 above)
Comment (SamuelHassine) [reply LastInfoSec/src/requirements.txt]: Agree.

File: LastInfoSec/src/lastinfosec.py (hunk 6 above)
Comment (SamuelHassine) [reply LastInfoSec/src/lastinfosec.py:82]: Agree.

File: LastInfoSec/Dockerfile (hunk 1 above)
Comment (remydewa) [reply LastInfoSec/Dockerfile]: Fixed :)

File: LastInfoSec/Dockerfile (hunk 2 above)
Comment (remydewa) [reply LastInfoSec/Dockerfile]: Fixed :)

File: LastInfoSec/docker-compose.yml (hunk 3 above)
Comment (remydewa) [reply LastInfoSec/docker-compose.yml]: Fixed :)

File: LastInfoSec/src/config.yml.sample (hunk 4 above)
Comment (remydewa) [reply LastInfoSec/src/config.yml.sample]: Fixed :)

File: LastInfoSec/src/requirements.txt (hunk 5 above)
Comment (remydewa) [reply LastInfoSec/src/requirements.txt]: Fixed :)

File: LastInfoSec/src/lastinfosec.py (hunk 6 above)
Comment (remydewa) [reply LastInfoSec/src/lastinfosec.py:82]: Fixed :)

File: LastInfoSec/docker-compose.yml [hunk 7]
```diff
@@ -1,15 +1,15 @@
-version: '3'
+version: '3.2'
 services:
-  connector-lastinfosec-cti:
-    image: connector-lastinfosec-cti:0.0.5
+  connector-lastinfosec:
+    image: connector-lastinfosec:0.0.5
```
Comment (rhaist) [inline LastInfoSec/docker-compose.yml]: This should be:
`    image: connector-lastinfosec:3.2.0`

Comment (fsnormand) [reply LastInfoSec/docker-compose.yml]: fixed :)

**Comment by fsnormand** [comment]
Hello ,
I tested your modifications on our connector. 
However, we have an error "dict could not use the encode function", because you deleted the json.dumps function to convert the dict to string. 
I added the json.dumps but you changed the bundle STIX sent function too and now we cannot integrate data in the platform.
Could you fix these issues or rollback with the self.api.stix2.import_bundle_from_json function ?

**Pull request merged**